cliquet/tests/test_storage.py

```diff
@@ -369,6 +369,17 @@ def test_get_all_can_filter_with_list_of_values(self):
                                           **self.storage_kw)
         self.assertEqual(len(records), 2)
 
+    def test_get_all_can_filter_with_list_of_numeric_values(self):
+        for l in [1, 10, 6, 46]:
+            self.create_record({'code': l})
+        sorting = [Sort('code', 1)]
+        filters = [Filter('code', 10, utils.COMPARISON.MAX)]
+        records, _ = self.storage.get_all(sorting=sorting, filters=filters,
+                                          **self.storage_kw)
+        self.assertEqual(records[0]['code'], 1)
+        self.assertEqual(records[1]['code'], 6)
+        self.assertEqual(records[2]['code'], 10)
```

Comment:
You're not testing that the length of the list of records is only 3 (and thus not returning 46)